apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx

```diff
@@ -64,6 +71,25 @@ export function BrowseCompany() {
     federalStates,
     isJobFair2022Participant,
   })
+  const { data: companyProfile } = useTpCompanyProfileQuery()
+  const tpCompanyProfileUpdateMutation = useTpCompanyProfileUpdateMutation()
+
+  const toggleFavorites = (value) => {
```

Comment:
Same -  can we pick a clearer name? I feel it's easy to confuse this function with toggleOnlyFavoritesFilter

Reply:
Done!